core/src/test/java/io/questdb/test/cutlass/http/ImportIODispatcherTest.java

```diff
@@ -450,7 +451,7 @@ public void testImportDesignatedTsFromSchema() throws Exception {
                     }
 
                     TableToken tableToken = new TableToken("syms", "syms", 0, false, false, false);
-                    try (TableReader reader = new TableReader(engine.getConfiguration(), tableToken)) {
+                    try (TableReader reader = new TableReader(0, engine.getConfiguration(), tableToken, engine.getTxnScoreboardPool())) {
```

Comment:
we should create off-pool readers consistently and with fewer parameters perhaps